index.js

```diff
@@ -21,15 +21,21 @@ const validationErrors = async (db, collectionName, { doc, err }) => {
 
 const explainSchemaErrors = (incomingDb, options = {}) => {
   const db = incomingDb;
-  const { onError } = options;
+  const { onError, includeValidationInError } = options;
   if (onError) {
     db.onValidationError = onError;
   }
-  const explainValidationError = async (...args) => {
+  const explainValidationErrorLogic = async (...args) => {
     const { valid, errors } = await validationErrors(...args);
-    if (!valid) {
+    if (!valid && db.onValidationError) {
```

Comment:
Added additional check here are it is now more possible that `onError` / `onValidationError` is not being used.